frontend/routes/auth.js

```diff
@@ -1,7 +1,7 @@
 const config = require('../config');
-const db = require('../utils/database');
+const db = require('../../datastore/utils/database');
```

Comment:
is this for the oauthServer?
perhaps the database dependency for that should be split and live in frontend?